tasks/copy_to_glacier/README.md

```diff
@@ -127,16 +176,17 @@ The `copy_to_glacier` lambda function expects that the input payload has a `gran
   }
 }
 ```
-
+From the json file, the `filepath` shows the current S3 location of files that need to be copied over to glacier ORCA S3 bucket such as `"filename": "s3://orca-sandbox-protected/MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065109.hdf"`.
 **Note:** We suggest that the `copy_to_glacier` task be placed any time after the `MoveGranulesStep`. It will propagate the input `granules` object as output, so it can be used as the last task in the workflow.
+See `schemas/input.json` for more information.
 
 
 ## Output
 
-The copy lambda will, as the name suggests, copy a file from its current some source destination. The destination location is defined as 
+The copy lambda will, as the name suggests, copy a file from its current source destination. The destination location is defined as 
 `${glacier_bucket}/${filepath}`, where `${glacier_bucket}` is pulled from your Cumulus `meta.bucket` config and `${filepath}` is pulled from the Cumulus granule object input.
```

Comment:
I don't believe this is correct anymore. This is pulled from an environment variable

Reply:
changed to 

The destination location is defined as 
`${glacier_bucket}/${filepath}`, where `${glacier_bucket}` is pulled from the environment variable `ORCA_DEFAULT_BUCKET` and `${filepath}` is pulled from the Cumulus granule object input.